src/manipulation.js

```diff
@@ -27,7 +27,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:-]+)[^>]*)\/>/gi,
-	rnoInnerhtml = /<(?:script|style|link)/i,
+	rnoInnerhtml = /<script|<style|<link/i,
```

Comment:
Hm, i wonder, do we have similar regexps? Would be helpful to have a comment, which would point to the microsoft issue and perf suite